institution/views.py

```diff
@@ -63,8 +63,11 @@ def get_queryset(self):
         queries.extend(
             query_utils.create_queries_by_lookup_field(self.request_data, [
                 'institution_name', 'editors_notes', 'institution_city', 'institution_country',
-                'resource', 'change_user', 'institution_id'
-            ])
+                'change_user', 'institution_id', 'resources', 'images'
+            ], search_fields_maps={'resources': ['resources__resource_name', 'resources__resource_details',
+                                                 'resources__resource_url'],
+                                   'images': ['images__image_filename', 'images__thumbnail',
+                                              'images__licence_details']})
```

Comment:
as your mention, we can query m2m field by `Model.objects.filter(resources__resource_name='aaa')`


is it Django also support m2m field in F function `F('resources__resource_name')`?